.github/workflows/ci.yml

```diff
@@ -27,9 +27,47 @@ jobs:
         os: [macos-10.15, macos-11, macos-12]
     runs-on: ${{ matrix.os }}
     steps:
-     - uses: actions/checkout@v2
-     - name: Build Userspace
-       run: bazel build --apple_generate_dsym -c opt :release --define=SANTA_BUILD_TYPE=adhoc
+      - uses: actions/checkout@v2
+      - name: Build Userspace
+        run: bazel build --apple_generate_dsym -c opt :release --define=SANTA_BUILD_TYPE=adhoc
+
+  start_vm:
+    runs-on: e2e-host
+    steps:
+      - uses: actions/checkout@v2
+      - name: Start VM
+        run: |
+          pip3 install --user google-cloud-storage==2.5.0
+          python3 Testing/integration/actions/start-vm.py macOS_12.bundle.tar.gz
+
+  integration:
+    runs-on: e2e-vm
+    env:
+      VM_PASSWORD: ${{ secrets.VM_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install configuration profile
+        run: bazel run //Testing/integration:install_profile -- Testing/integration/configs/default.mobileconfig
+      - name: Add homebrew to PATH
+        run: echo "/opt/homebrew/bin/" >> $GITHUB_PATH
+      - name: Install and start moroz
+        run: |
+          go install github.com/kallsyms/moroz/cmd/moroz@d0bba3f4
```

Comment:
Are we able to upstream these moroz changes? If not we need to find a better place to host them. The external_patches dir + a bazel build rule for them may do?

https://github.com/google/santa/tree/main/external_patches

Reply:
Doesn't seem like these will be able to be upstreamed on any regular basis, so rewrote with a patch.